src/data/dfinsupp/order.lean

```diff
@@ -59,22 +57,22 @@ variables [Π i, preorder (α i)]
 instance : preorder (Π₀ i, α i) :=
 { le_refl := λ f i, le_rfl,
   le_trans := λ f g h hfg hgh i, (hfg i).trans (hgh i),
-  .. dfinsupp.has_le α }
+  .. dfinsupp.has_le }
 
 lemma coe_fn_mono : monotone (coe_fn : (Π₀ i, α i) → Π i, α i) := λ f g, le_def.1
 
 end preorder
 
 instance [Π i, partial_order (α i)] : partial_order (Π₀ i, α i) :=
 { le_antisymm := λ f g hfg hgf, ext $ λ i, (hfg i).antisymm (hgf i),
-  .. dfinsupp.preorder α}
+  .. dfinsupp.preorder }
 
 instance [Π i, semilattice_inf (α i)] : semilattice_inf (Π₀ i, α i) :=
 { inf := zip_with (λ _, (⊓)) (λ _, inf_idem),
   inf_le_left := λ f g i, by { rw zip_with_apply, exact inf_le_left },
   inf_le_right := λ f g i, by { rw zip_with_apply, exact inf_le_right },
   le_inf := λ f g h hf hg i, by { rw zip_with_apply, exact le_inf (hf i) (hg i) },
-  ..dfinsupp.partial_order α }
+  ..dfinsupp.partial_order }
 
 @[simp] lemma inf_apply [Π i, semilattice_inf (α i)] (f g : Π₀ i, α i) (i : ι) :
   (f ⊓ g) i = f i ⊓ g i :=
```

Comment:
The changes to this lemma are here to fix the implicitness of `α` to this lemma.